backend/manager/modules/restapi/types/src/main/java/org/ovirt/engine/api/restapi/types/VmMapper.java

```diff
@@ -291,7 +291,7 @@ public static Vm map(org.ovirt.engine.core.common.businessentities.VM entity, Vm
                 Boot boot = map(entity.getBootSequence(), null);
                 model.getOs().setBoot(boot);
             }
-            if (VmCpuCountHelper.isAutoPinning(entity)) {
+            if (VmCpuCountHelper.isResizeAndPinPolicy(entity)) {
```

Comment:
Shouldn't it be isDynamicPinPolicy? Otherwise the data for dedicated CPUs wont be sent.

Reply:
only the cpu pinning part.